ClearKeep/Scenes/Chat/Views/ChatView.swift

```diff
@@ -657,16 +659,24 @@ private extension ChatView {
 	
 	func loadLocalMessage() {
 		messages = injected.interactors.chatInteractor.getMessageFromLocal(groupId: groupId)
-
+		if messages?.count ?? 0 < 20 {
+			isEndOfPage = true
+		}
+		messages?.forEach({ message in
+			dataMessages.append(MessageViewModel(data: message, members: group?.groupMembers ?? []))
+		})
 		notificationToken = messages?.observe({ changes in
 			switch changes {
 			case .initial:
-				if messages?.count ?? 0 < 20 {
-					isEndOfPage = true
-				}
-				messages?.forEach({ message in
-					dataMessages.append(MessageViewModel(data: message, members: group?.groupMembers ?? []))
-				})
+                if dataMessages.count > 0 {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/0b09249675dff5b0a460b69f27e9f3e1d518728a/ClearKeep/Scenes/Chat/Views/ChatView.swift#L671"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>ChatView.swift:671</code></td>
    </tr>
  </tbody>
</table>